### PR TITLE
Express package installed for the app

### PR DESCRIPTION
**Pull request by vijaykumar-evenion**
On branch develop
Changes to be committed:
	new file:   .gitignore
	new file:   package-lock.json
	modified:   package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,5 +8,8 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.18.2"
```
Comment (svk-vijaykumar) [inline package.json:13]: Along with this install dotenv package

Comment (svk-vijaykumar) [inline package.json:13]: Make you are using correct express version

**Review by svk-vijaykumar — COMMENTED** [review]
Along with this install dotenv package

File: package.json (hunk 1 above)
Comment (vijaykumar-evenion) [reply package.json:13]: Yes we are using correct & latest version of express

**Review by svk-vijaykumar — APPROVED** [review]
Everything looking good

**Pull request merged**